extension/src/experiments/columns/constants.ts

```diff
@@ -11,3 +11,12 @@ export const timestampColumn: Column = {
 }
 
 export const EXPERIMENT_COLUMN_ID = 'id'
+
+export const COMMIT_COLUMN_ID = 'commit'
+export const BRANCH_COLUMN_ID = 'branch'
+
+export const DEFAULT_COLUMN_IDS = [
```

Comment:
Added two more default columns (ids are commt and branch). These always stay in the order on the backend and appear on the frontend if the table is sorted. Similar to the `id` column, they are not draggable and can't be unselected.